app/__init__.py

```diff
@@ -0,0 +1,27 @@
+# app/__init__.py
+from flask import Flask
+from flask_api import FlaskAPI
+from flask_sqlalchemy import SQLAlchemy
+
+from flask import request, jsonify, abort
```

Comment:
You don't seem to be using any of these in the file

Reply:
I have remove:
`from flask_api import FlaskAPI` and `from flask import request, jsonify, abort`
But I am using `from flask import Flask` and `from flask_sqlalchemy import SQLAlchemy` for the app set up and db.